hcl/fmt/fmt.go

```diff
@@ -65,6 +66,7 @@ func FormatTree(dir string) ([]FormatResult, error) {
 	if err != nil {
 		return nil, errors.E(errFormatTree, err)
 	}
+	sort.Strings(files)
```

Comment:
The other API clients of the `ListTerramateFiles()` and `ListTerramateDirs()` do not require a specific ordering, as the files are used to build the configuration tree or check changed files, etc.